rails/spec/javascript/components/Sort.spec.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Sort from '../../../app/javascript/components/Sort';
+
+describe('Sort component', () => {
+  beforeEach(() => {
+    global.I18n = {
+      t: jest.fn(),
+    };
+  });
+
+  const corinneStory = {
+    created_at: '2020-10-03T11:13:23.037Z',
+    title: "Corinne's testimonial",
+  };
+
+  const rudoStory = {
+    created_at: '2020-10-01T11:13:23.013Z',
+    title: "Rudo's testimonial",
+  };
+
+  it('Displays correctly', () => {
+    const wrapper = shallow(<Sort stories={[corinneStory, rudoStory]} />);
+
+    expect(wrapper).toMatchSnapshot();
+
+    expect(global.I18n.t).toHaveBeenCalledWith('sort_stories');
+    expect(global.I18n.t).toHaveBeenCalledWith('most_recent');
+    expect(global.I18n.t).toHaveBeenCalledWith('alphabetical');
+    expect(global.I18n.t).toHaveBeenCalledWith('reversed_alphabetical');
```

Comment:
What do you think of moving out these hardcoded strings into a const array? Then we can easily update the text or add to it if the Sort menu changes. You could define an array of sortValues which have all these strings in them and then use a for loop to call the expect statement on each.